db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).pluck(:id).each do |id|
```

Comment:
Thinking on this further - wrapping this whole thing in a `Attachment.transaction` block so all of those update statements are batched and enlisted into a single transaction/roundtrip should run much quicker.